developer_manual/core/ocs-share-api.rst

```diff
@@ -159,14 +207,13 @@ Get all pending federated cloud shares the user has received.
 Statuscodes:
 
 * 100 - successful
-* 404 - couldn't fetch shares
 
 Accept a pending Federated Cloud Share
 --------------------------------------
 
 Locally accept a received federated cloud share that was sent from a remote instance.
 
-* Syntax: /remote_shares/*<share_id>*
+* Syntax: /remote_shares/pending/*<share_id>*
```

Comment:
When getting all pending remote shares there are two fields, id and remote_id.
Here this field share_id is correlated with remote_id or id?